# HG changeset patch
# User Carlo de Falco <kingcrimson@tiscali.it>
# Date 1315322049 -7200
#      Tue Sep 06 17:14:09 2011 +0200
# Node ID 3b5449493bb8907ebc32eade82d09f247064458d
# Parent  2e2738837a240d96675afb098a2191eaef73b4e8
Remove console output in graphics demos.

* loglog.m, semilogx.m,semilogy.m: remove console
output from demos.

diff --git a/scripts/plot/loglog.m b/scripts/plot/loglog.m
--- a/scripts/plot/loglog.m
+++ b/scripts/plot/loglog.m
@@ -75,19 +75,16 @@ endfunction
 %! subplot (1, 2, 1)
 %! loglog (a, b)
 %! xlabel ('loglog (a, b)')
 %! 
 %! subplot (1, 2, 2)
 %! loglog (a, abs (b))
 %! set (gca, 'ydir', 'reverse')
 %! xlabel ('loglog (a, abs (b))')
-%! 
-%! fprintf ('the two plots should look the same except for the sign in y tick labels')
-
 
 %!test
 %! hf = figure (1232, "visible", "off");
 %! unwind_protect  
 %!   a = logspace (-5, 1, 10);
 %!   b = logspace (-5, 1, 10);
 %!   loglog (a, b)
 %!   assert (get (gca, "yscale"), "log");
diff --git a/scripts/plot/semilogx.m b/scripts/plot/semilogx.m
--- a/scripts/plot/semilogx.m
+++ b/scripts/plot/semilogx.m
@@ -73,18 +73,16 @@ endfunction
 %! subplot (1, 2, 1)
 %! semilogx (b, a)
 %! xlabel ('semilogx (a, b)')
 %! 
 %! subplot (1, 2, 2)
 %! semilogx (abs (b), a)
 %! set (gca, 'ydir', 'reverse')
 %! xlabel ('semilogx (a, abs (b))')
-%! 
-%! fprintf ('the two plots should look the same except for the sign in x tick labels')
 
 %!test
 %! hf = figure (1232, "visible", "off");
 %! unwind_protect  
 %!   a = logspace (-5, 1, 10);
 %!   b = logspace (-5, 1, 10);
 %!   semilogx (a, b)
 %!   assert (get (gca, "xscale"), "log");
diff --git a/scripts/plot/semilogy.m b/scripts/plot/semilogy.m
--- a/scripts/plot/semilogy.m
+++ b/scripts/plot/semilogy.m
@@ -74,18 +74,16 @@ endfunction
 %! subplot (1, 2, 1)
 %! semilogy (a, b)
 %! xlabel ('semilogy (a, b)')
 %! 
 %! subplot (1, 2, 2)
 %! semilogy (a, abs (b))
 %! set (gca, 'ydir', 'reverse')
 %! xlabel ('semilogy (a, abs (b))')
-%! 
-%! fprintf ('the two plots should look the same except for the sign in y tick labels')
 
 %!test
 %! hf = figure (1232, "visible", "off");
 %! unwind_protect  
 %!   a = logspace (-5, 1, 10);
 %!   b = logspace (-5, 1, 10);
 %!   semilogy (a, b)
 %!   assert (get (gca, "yscale"), "log");
